composer.json

```diff
@@ -38,10 +41,10 @@
       "vagrant ssh -c \"cd /vagrant/content/plugins/core-sitemaps && WP_TESTS_DB_PASS=password composer run test:phpunit\""
     ],
     "test:phpcs": [
-      "vendor/bin/phpcs -nps --colors --report-code --report-summary --report-width=80 ."
+      "phpcs"
```

Comment:
I'd expect us to use a vendor version here to ensure everyone is using the same version of phpcs, probably something like `vendor/squizlabs/php_codesniffer/bin/phpcs --standard=./phpcs.xml.dist`.

Reply:
The test:phpcs command is effectively unchanged compared to before this commit, all I've done is move all the parameters inside the phpcs.xml.dist file, which is the file called by phpcs when none is specified.

I simplified the configuration based on the best practices suggested in https://dev.hmn.md/2019/10/10/making-the-best-of-config/.

This can be verified by changing the line  to `which phpcs` it displays _/Users/sander/dev/_hm/core-sitemaps/content/plugins/core-sitemaps/vendor/bin/phpcs_  as for composer this is an equivalent command.